draft-ietf-spice-sd-cwt.md

```diff
@@ -539,34 +548,34 @@ All other claims are OPTIONAL in an SD-KBT.
 # SD-KBT and SD-CWT Verifier Validation
 
 The exact order of the following steps MAY be changed, as long as all checks are performed before deciding if an SD-CWT is valid.
-First the Verifier must open the protected headers of the SD-KBT and find the issuer SD-CWT present in the `kcwt` field.
-Next, the Verifier must validate the SD-CWT as described in {{Section 7.2 of !RFC8392}}.
-The Verifier extracts the confirmation key from the `cnf` claim in the SD-CWT payload.
-Using the confirmation key, the Verifier validates the SD-KBT as described in {{Section 7.2 of !RFC8392}}.
 
-Finally, the Verifier MUST extract and decode the disclosed claims from the `sd_claims` header parameter in the unprotected header of the SD-CWT.
-The decoded `sd_claims` are converted to an intermediate data structure called a Digest To Disclosed Claim Map that is used to transform the Presented Disclosed Claims Set into a Validated Disclosed Claims Set.
-The Verifier MUST compute the hash of each Salted Disclosed Claim (`salted`), in order to match each disclosed value to each entry of the Presented Disclosed Claims Set.
+{::options nested_ol_types="1, a, i" /}
 
-One possible concrete representation of the intermediate data structure for the Digest To Disclosed Claim Map could be:
+ 1. First the Verifier must open the protected headers of the SD-KBT and find the issuer SD-CWT present in the `kcwt` field.
 
-~~~ cddl-ish
-{
-  &(digested-salted-disclosed-claim) => salted
-}
-~~~
+ 2. Next, the Verifier must validate the SD-CWT as described in {{Section 7.2 of !RFC8392}}.
+
+ 3. The Verifier extracts the confirmation key from the `cnf` claim in the SD-CWT payload.
 
-The Verifier constructs an empty cbor map called the Validated Disclosed Claims Set, and initializes it with all mandatory to disclose claims from the verified Presented Disclosed Claims Set.
-Next, the Verifier performs a breadth first or depth first traversal of the Presented Disclosed Claims Set and Validated Disclosed Claims Set, using the Digest To Disclosed Claim Map to insert claims into the Validated Disclosed Claims Set when they appear in the Presented Disclosed Claims Set.
+ 4. Using the confirmation key, the Verifier validates the SD-KBT as described in {{Section 7.2 of !RFC8392}}.
+
+ 5. Finally, the Verifier MUST extract and decode the disclosed claims from the `sd_claims` header parameter in the unprotected header of the SD-CWT.
+    The decoded `sd_claims` are converted to an intermediate data structure called a Digest To Disclosed Claim Map that is used to transform the Presented Disclosed Claims Set into a Validated Disclosed Claims Set.
+    The Verifier MUST compute the hash of each Salted Disclosed Claim (`salted`), in order to match each disclosed value to each entry of the Presented Disclosed Claims Set.
+    One possible concrete representation of the intermediate data structure for the Digest To Disclosed Claim Map could be: `{ &(digested-salted-disclosed-claim) => salted }`
```

Comment:
I don't understand the `&` notation used here.  I suspect other readers won't either.

Reply:
Then explicitly that it's CDDL.